packages/aa-comunica-parser/lib/ComunicaAlgebraTransformer.ts

```diff
@@ -0,0 +1,38 @@
+import { toAlgebra12Builder } from '@traqula/algebra-sparql-1-2';
+import { translateGraphPattern, accumulateGroupGraphPattern } from '@traqula/algebra-transformations-1-1';
+import type { AlgebraIndir, ContextConfigs } from '@traqula/algebra-transformations-1-1';
+import type { Algebra } from '@traqula/algebra-transformations-1-2';
+import { createAlgebraContext } from '@traqula/algebra-transformations-1-2';
+import { IndirBuilder } from '@traqula/core';
+import type * as T11 from '@traqula/rules-sparql-1-1';
```

Comment:
I see some usages of 1.1 below.
For the sake of maintainability and clarify for future SPARQL versions, would it make sense to re-export all typings from the 1.2 package?
So that all 1.1 typings also become 1.2 typings, and developers only need to be aware of the latest SPARQL version (being currently 1.2).